src/main/java/svc/dialects/ImprovedHSQLDialect.java

```diff
@@ -0,0 +1,62 @@
+package svc.dialects;
+
+import org.hibernate.boot.Metadata;
+import org.hibernate.dialect.HSQLDialect;
+import org.hibernate.mapping.Table;
+import org.hibernate.tool.schema.spi.Exporter;
+/*
+ *  Removes HSQL Errors
+ *  See: https://stackoverflow.com/questions/12054422/unsuccessful-alter-table-xxx-drop-constraint-yyy-in-hibernate-jpa-hsqldb-standa
+ */
+public class ImprovedHSQLDialect extends HSQLDialect {
```

Comment:
Can you explain a little about what this is for in more detail? I've never seen this before and haven't yet encountered a need for this in other projects.

Reply:
I was getting the same error message as in: https://stackoverflow.com/questions/12054422/unsuccessful-alter-table-xxx-drop-constraint-yyy-in-hibernate-jpa-hsqldb-standa

If I remember right, the error didn't stop the code from running but it wasn't pretty on the console and this code cleans it up.